src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/colors.scss

```diff
@@ -12,3 +12,4 @@ $silver: #ccc;
 $silverChalice: #a2a2a2;
 $white: #fff;
 $wildSand: #f5f5f5;
+$alto: #d2d2d2;
```

Comment:
Please order the colors alphabetically. I already realized that stylelint doesn't check for double variable definitions, and if they are ordered in some way, they would be easier to detect.